src/Quasar/Mount/MongoDB.purs

```diff
@@ -79,7 +80,7 @@ toURI { hosts, auth, props } =
         (URI.Authority
           (combineCredentials <<< _.credentials <<< unwrap <$> auth)
           (oneOf hosts)))
-      (_.path <<< unwrap <$> auth))
```

Comment:
How do you like

`Just (fromMaybe (Left P.rootDir) (_.path <<< unwrap <$> auth))`

over?

`maybe (Just (Left P.rootDir)) Just (_.path <<< unwrap <$> auth))`